src/components/order/Order.jsx

```diff
@@ -32,28 +32,17 @@ const CloseButton = styled(Button)`
     font-size: 1.5rem;
   }
 `
-// #endregion
+//#endregion
```

Comment:
Where is this style comingfrom? What does this help with? Something inside vscode?

Reply:
I believe this originates from C# but is now also present in VSCode editor for multiple languages.
It lets you define a region that you can then fold, makes it easier to work with larger files.
https://code.visualstudio.com/docs/editor/codebasics#_folding

Reply:
Ah cool, thanks for feedback!